benefits/enrollment/templates/enrollment/success.html

```diff
@@ -7,26 +7,45 @@
   {{ block.super |add:" enrollment-success" }}
 {% endblock classes %}
 
+{% block page_title %}
+  {% translate "enrollment.pages.success.title" %}&nbsp;|&nbsp;
+{% endblock page_title %}
+
 {% block headline %}
-  <div class="col-lg-10">
-    <h1 class="pb-lg-8 pb-4">{{ page.headline }}</h1>
+  <div class="col-lg-8">
+    <h1 class="pb-lg-5 pb-4">{% translate "enrollment.pages.success.headline" %}</h1>
   </div>
 {% endblock headline %}
 
 {% block inner-content %}
-  <div class="col-12 col-sm-12 col-lg-10">{% include "core/includes/media-list.html" with media=media %}</div>
+  <div class="col-12 col-sm-12 col-lg-9">
+    <div class="row flex-column-reverse flex-lg-row">
+      <div class="col-12 col-lg-6">
+        <p class="pt-lg-4 mt-lg-3">{% translate "enrollment.pages.success.login_gov.confirm_item.details" %}</p>
```

Comment:
Is this correct? Either the `msgid` should be updated to be more general, or this is hardcoding the Login.gov specific text here?

Reply:
This PR does not entirely re-do the success.html template to the new style. This PR merely removes as much viewmodel as possible (media item, page for title, headline), but still uses a viewmodel for the Log Out button. Since the body copy is the same across all 3 flows for now (with all the [brackets]), the success.html template points to the same 1 copy for now. Once we have differing copy for various flows (like, maybe, SacRT-Senior), this template can be refactored to show different body copy per flow. So the `msgid` can be updated for the time being until we get the new copy. It's sort of a placeholder until we get flow-specific enrollment success body copy in the meantime.

Should I do this instead:
- Delete `"enrollment.pages.success.veteran.confirm_item.details"`, `"enrollment.pages.success.login_gov.confirm_item.details"`, `"enrollment.pages.success.mst_cc.confirm_item.details"`.
- Create and use generic `"enrollment.pages.success.confirm_item.details"`
- In future PRs, once we have flow-specific copy, change copy/template as needed.

Reply:
> Should I do this instead:

Yes that all sounds good. But also, remove this field from the models for now and use it directly in the template.

We will revisit for CalFresh, but not for a little while.

Reply:
Got it! It may have to be revisited when SacRT launches cos SacRT is only light rail, not bus.